### PR TITLE
jet 0.13.0 (new formula)

### PR DESCRIPTION
**Pull request by gregstewart**
Add Codeship's CLI for their Docker infrastructure

### CODE REVIEW

File: Library/Formula/jet.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class Jet < Formula
+  desc "CLI for Codeships Docker infrastructure"
+  homepage "https://codeship.com/documentation/docker/release-notes/"
+  url "https://s3.amazonaws.com/codeship-jet-releases/0.13.0/jet-darwin_amd64_0.13.0.tar.gz"
```
Comment (bfontaine) [inline Library/Formula/jet.rb]: We need a source release here.

Comment (gregstewart) [inline Library/Formula/jet.rb]: I don't believe there's a source release. I should use a `bottle` then shouldn't I?

Comment (bfontaine) [inline Library/Formula/jet.rb]: Bottles are built by our CI bot. If there’s no source release it should go to our [`homebrew/binary`](https://github.com/Homebrew/homebrew-binary) tap.

Comment (gregstewart) [inline Library/Formula/jet.rb]: No worries, I will move it over there then.

**Comment by gregstewart** [comment]
Opened Homebrew/homebrew-binary#291.

**Comment by bfontaine** [comment]
Thanks!

**Pull request closed without merging**